clicks.view.lkml

```diff
@@ -89,14 +121,17 @@ view: clicks {
 
 
   # Page
+  # This section contains fields that are duplicated across over view files in this project
 
   dimension: page_url {
+    description: "The web page URL."
     type: string
     sql: ${TABLE}.page_url ;;
     group_label: "Page"
   }
 
   dimension: node_id {
+    description: "The Webtrends node identifier."
```

Comment:
This has nothing to do with Webtrends. This is from CMS lite. This is the page identifier for www2.gov.bc.ca. We are using this to connect the page hit to the meta data in the csv. Also will end up being used to display page analytics. Not all sites will have a node_id

Reply:
Agreed, that was a mistake; thanks for catching it. I will update the description to "The CMS Lite node identifier for Gov pages."